index.html

```diff
@@ -901,6 +901,16 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 <li>Return <code>fail</code>.</li>
 </ol>
 </section>
+<section id="sec-iana-considerations">
+<h4>IANA Considerations</h4>
+<p>This section registers <a>Timing-Allow-Origin</a> as a <a href="https://tools.ietf.org/html/rfc3864#section-4.2.2">Provisional Message Header</a>.</p>
+<dl>
+<dt>Header field name:</dt><dd><pre class="abnf">Timing-Allow-Origin</pre></dd>
+<dt>Applicable protocol:</dt><dd><a href="https://tools.ietf.org/html/rfc2616">http</a></dd>
+<dt>Status:</dt><dd>standard</dd>
```

Comment:
Should it be "provisional" or "standard"?

Reply:
If this is a Recommendation, standard.

Reply:
RT is still in CR, so I modified it to "provisional".

Reply:
yes, but it's on the recommendation track.

Reply:
> yes, but it's on the recommendation track.

so it should be "standard"?

Reply:
Yes.